assets/public/css/styles.css

```diff
@@ -0,0 +1,304 @@
+@font-face {
+  font-family: "axis";
+  src: url("../fonts/axis.otf");
+}
+@font-face {
+  font-family: "Montserrat-Bold";
+  src: url("../fonts/Montserrat-Bold.ttf");
+}
+@font-face {
+  font-family: "Montserrat-ExtraBold";
+  src: url("../fonts/Montserrat-ExtraBold.ttf");
+}
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+a {
+  text-decoration: none;
+  color: var(--secondary-color);
+}
+
+button {
```

Comment:
try to avoid selecting all "button" elements at once

Reply:
These are defined in _resets.scss and are meant to remove / override default browser styles that don't match the design styling. Otherwise I would be writing redundant code again and again in all places to get the same styling property values.